code-challenges.test.js

```diff
@@ -67,3 +137,35 @@ const vowelTester3 = "challenges"
 
 
 // b) Create the function that makes the test pass.
+ 
+//Psuedo Code
+    // declare a function named vowelSearch
+    // use a for loop to iterate through each character in the string
+    // use conditional statments and .charAt to compare each letter of the word to vowels. 
+        //if the letter is "e" - return the index of "e"
+        //else if the letter is "a" - return the index of "a"
+        //else if the letter is "i" - return the index of "i"
+        //else if the letter is "o" - return the index of "o"
+        //else if the letter is "u" - return the index of "u"
+
+
+const vowelSearch =  (string) => {
+    for (let i=0 ; i < string.length ; i++) {
+        if (string.charAt(i) === "e"){
+            return string.indexOf("e")
+    } else if (string.charAt(i) === "a") { 
+            return string.indexOf("a")
+        } else if (string.charAt(i) === "i") {
+            return string.indexOf("i")
+        } else if (string.charAt(i) === "o"){
+            return string.indexOf("o")
+        } else if (string.charAt(i) === "u"){
+            return string.indexOf("u")
+        }
+    }
+}
+
+// console.log(vowelSearch(vowelTester1))
+// console.log(vowelSearch(vowelTester2))
+// console.log(vowelSearch(vowelTester3))
+
```

Comment:
Will, you are Awesome!   You crushed these problems!  Your tests and pseudocode are on point, and your code is clean.  The last one is tough, but I think by now you know the regex method search.  However, you did great with the material covered.  I encouarge you to refactor using search just to get used to using regex.  It will help you down the line.  The point is you understand how to do this the hard way. :) You continued to rock it this week!